src/ParetoSmooth.jl

```diff
@@ -16,6 +16,12 @@ function __init__()
 
 end
 
+if VERSION >= v"1.8"
```

Comment:
Maybe this fixes the Julia 1.6 test errors:
```suggestion
@static if VERSION >= v"1.8"
```

Reply:
I thought it only mattered if something wouldn't parse or was syntactically invalid in an older Julia version.